packages/cfd/src/Constants/jurisdiction-contents.ts

```diff
@@ -32,6 +33,13 @@ export const jurisdiction_contents: TJurisdictionContent = {
             )}`,
             `${localize('Leverage up to 1:1000')}`,
         ],
+        swapfree_contents: [
+            `${localize('Selecting this will onboard you through Deriv (SVG) LLC (company no. 273 LLC 2020)')}`,
+            `${localize(
+                '40+ assets: forex (standard/micro), stocks, stock indices, commodities, cryptocurrencies, and ETFs swap-free.'
+            )}`,
+            `${localize('Leverage up to 1:1000')}`,
```

Comment:
```suggestion
            `${localize('Selecting this will onboard you through Deriv (SVG) LLC (company no. 273 LLC 2020)')}`,
            `${localize(
                '40+ assets: forex (standard/micro), stocks, stock indices, commodities, cryptocurrencies, and ETFs swap-free.'
            )}`,
            `${localize('Leverage up to 1:1000')}`,
```
Nit: This is a pretty weird way to use localize. To put it inside an array you can do it using the <Localize /> component instead

Reply:
Its a constant value used in a component already :)